### PR TITLE
chore: edit card variables

### PR DESCRIPTION
**Pull request by zhpenkov**
Add group variables for the **Card** component for [ **bootstrap, default, classic, material, fluent** ] theme.

### CODE REVIEW

File: packages/bootstrap/scss/card/_variables.scss [hunk 1]
```diff
@@ -1,71 +1,184 @@
...
+/// The text color of the Card when focused.
+/// @group card
 $kendo-card-focus-text: null !default;
+/// The border color of the Card when focused.
+/// @group card
 $kendo-card-focus-border: null !default;
+
+// TODO: double-check: the default theme is not supposed to elevate the Card on focus
+
+/// The shadow of the Card when focused.
```
Comment (epetrow) [inline packages/bootstrap/scss/card/_variables.scss:59]: The shadow of the Card when focused > The shadow of the focused Card

**Review by epetrow — COMMENTED** [review]
A few things I've noticed. I left a comment only on one of the `_variables.scss` files as to not be repetitive, however they should be addressed in all files.

File: packages/bootstrap/scss/card/_variables.scss [hunk 2]
```diff
@@ -1,71 +1,184 @@
...
+/// @group card
 $kendo-card-footer-bg: $kendo-card-header-bg !default;
+/// The text color of the Card footer.
+/// @group card
 $kendo-card-footer-text: $kendo-card-header-text !default;
+/// The border color of the Card footer.
+/// @group card
 $kendo-card-footer-border: $kendo-card-header-border !default;
 
+/// The bottom margin of the Card subtitle.
```
Comment (inikolova) [inline packages/bootstrap/scss/card/_variables.scss:108]: I think here should be /// The bottom margin of the Card **title**. (please double check)

File: packages/bootstrap/scss/card/_variables.scss [hunk 3]
```diff
@@ -1,71 +1,184 @@
...
+/// @group card
 $kendo-card-title-margin-bottom: k-map-get( $kendo-spacing, 3 ) !default;
+/// The text size of the Card title.
```
Comment (inikolova) [inline packages/bootstrap/scss/card/_variables.scss:111]: The text size of the Card title. -> Should be The **font** size of the Card title. (for consistency)

File: packages/bootstrap/scss/card/_variables.scss [hunk 4]
```diff
@@ -1,71 +1,184 @@
...
+/// @group card
 $kendo-card-title-font-size: $kendo-h5-font-size !default;
+/// The font family of the Card title.
+/// @group card
 $kendo-card-title-font-family: $headings-font-family !default;
+/// The line height of the Card title.
+/// @group card
 $kendo-card-title-line-height: $headings-line-height !default;
+/// The text weight of the Card title.
```
Comment (inikolova) [inline packages/bootstrap/scss/card/_variables.scss:120]: The text weight... -> The **font** weight... (for consistency)

File: packages/bootstrap/scss/card/_variables.scss [hunk 5]
```diff
@@ -1,71 +1,184 @@
...
+/// @group card
 $kendo-card-subtitle-letter-spacing: null !default;
+/// The text color of the Card subtitle.
+/// @group card
 $kendo-card-subtitle-text: $kendo-subtle-text !default;
 
+/// The maximum width of the Card image.
+/// @group card
 $kendo-card-img-max-width: 100px !default;
+/// The size of the avatar in the Card.
```
Comment (inikolova) [inline packages/bootstrap/scss/card/_variables.scss:152]: I would suggest to make the first letter of Avatar uppercase

File: packages/bootstrap/scss/card/_variables.scss [hunk 6]
```diff
@@ -1,71 +1,184 @@
...
+/// @group card
 $kendo-card-avatar-size: 45px !default;
+/// The spacing between the avatar and the text in the Card.
```
Comment (inikolova) [inline packages/bootstrap/scss/card/_variables.scss:155]: Same as the above

File: packages/bootstrap/scss/card/_variables.scss [hunk 7]
```diff
@@ -1,71 +1,184 @@
...
+/// @group card
 $kendo-card-avatar-spacing: $kendo-card-header-padding-x !default;
 
+/// The horizontal padding of the Card actions.
+/// @group card
 $kendo-card-actions-padding-x: k-map-get( $kendo-spacing, 2 ) !default;
+/// The vertical padding of the Card actions.
+/// @group card
 $kendo-card-actions-padding-y: k-map-get( $kendo-spacing, 2 ) !default;
+/// The top border of the Card actions.
```
Comment (inikolova) [inline packages/bootstrap/scss/card/_variables.scss:165]: Maybe here should be "The border width of the Card actions" or "The top border width of the Card actions" (for consistency)

**Comment by inikolova** [comment]
My suggestions should be applied to all of the themes. Also, please check the variables descriptions in the Fluent theme too and update them to match the descriptions in the other themes (where possible)

File: packages/fluent/scss/card/_variables.scss [hunk 8]
```diff
@@ -2,176 +2,180 @@
...
+/// The border color of the focused Card.
 /// @group card
 $kendo-card-focus-border: var( --kendo-component-border, initial ) !default;
-/// Shadow of the card when focused.
+/// The shadow of the focused Card.
 /// @group card
 $kendo-card-focus-shadow: var( --kendo-box-shadow-depth-3, none ) !default;
 
-/// Text size of the card header.
+/// The text size of the Card header.
```
Comment (epetrow) [inline packages/fluent/scss/card/_variables.scss:60]: font size instead of the text size here

File: packages/fluent/scss/card/_variables.scss [hunk 9]
```diff
@@ -2,176 +2,180 @@
...
 /// @group card
 $kendo-card-header-font-size: var( --kendo-font-size-lg, inherit ) !default;
-/// Font family of the card header text.
+/// the font family of the Card header text.
 /// @group card
 $kendo-card-header-font-family: var( --kendo-font-family, inherit ) !default;
-/// Text weight of the card header.
+/// The text weight of the Card header.
```
Comment (epetrow) [inline packages/fluent/scss/card/_variables.scss:66]: font weight instead of text weight

File: packages/bootstrap/scss/card/_variables.scss [hunk 10]
```diff
@@ -1,71 +1,184 @@
...
+/// The text color of the focused Card.
+/// @group card
 $kendo-card-focus-text: null !default;
+/// The border color of the focused Card.
+/// @group card
 $kendo-card-focus-border: null !default;
+
+// TODO: double-check: the default theme is not supposed to elevate the Card on focus
+
+/// The shadow of the focused Card
```
Comment (inikolova) [inline packages/bootstrap/scss/card/_variables.scss:59]: A missing dot

File: packages/fluent/scss/card/_variables.scss [hunk 11]
```diff
@@ -2,176 +2,180 @@
...
-/// Line height of the card title.
+/// The line height of the Card title.
 /// @group card
 $kendo-card-title-line-height: var( --kendo-line-height, normal ) !default;
-/// Bottom margin of the card title.
+/// The bottom margin of the Card title.
 /// @group card
 $kendo-card-title-margin-bottom: map.get( $kendo-spacing, 3 ) !default;
-/// Font size of the card subtitle.
+/// The font size of the Card title.
```
Comment (inikolova) [inline packages/fluent/scss/card/_variables.scss:132]: Maybe here should be The font size of the Card **subtitle**. -> please check the variable name

File: packages/fluent/scss/card/_variables.scss [hunk 12]
```diff
@@ -2,176 +2,180 @@
...
 /// @group card
 $kendo-card-subtitle-font-size: var( --kendo-font-size-sm, inherit ) !default;
-/// Font family of the card subtitle.
+/// The font family of the Card title.
```
Comment (inikolova) [inline packages/fluent/scss/card/_variables.scss:135]: The font family of the Card **subtitle**. -> please check the variable name

File: packages/fluent/scss/card/_variables.scss [hunk 13]
```diff
@@ -2,176 +2,180 @@
...
 /// @group card
 $kendo-card-subtitle-font-family: var( --kendo-font-family, inherit ) !default;
-/// Line height of the card subtitle.
+/// The line height of the Card title.
```
Comment (inikolova) [inline packages/fluent/scss/card/_variables.scss:138]: Same as the abpve

File: packages/fluent/scss/card/_variables.scss [hunk 14]
```diff
@@ -2,176 +2,180 @@
...
 /// @group card
 $kendo-card-subtitle-line-height: var( --kendo-line-height, normal ) !default;
-/// Bottom margin of the card subtitle.
+/// The font weight of the Card title.
```
Comment (inikolova) [inline packages/fluent/scss/card/_variables.scss:141]: Same as the above

File: packages/fluent/scss/card/_variables.scss [hunk 15]
```diff
@@ -2,176 +2,180 @@
...
 /// @group card
 $kendo-card-subtitle-margin-bottom: map.get( $kendo-spacing, 3 ) !default;
-/// Text color of the card subtitle.
+/// The letter spacing of the Card title.
```
Comment (inikolova) [inline packages/fluent/scss/card/_variables.scss:144]: Same as the above

File: packages/fluent/scss/card/_variables.scss [hunk 16]
```diff
@@ -2,176 +2,180 @@
...
-/// Font family of the card subtitle.
+/// The font family of the Card subtitle.
 /// @group card
 $kendo-card-subtitle-font-family: var( --kendo-font-family, inherit ) !default;
-/// Line height of the card subtitle.
+/// The line height of the Card subtitle.
 /// @group card
 $kendo-card-subtitle-line-height: var( --kendo-line-height, normal ) !default;
-/// Bottom margin of the card subtitle.
+/// The font weight of the Card subtitle.
```
Comment (inikolova) [inline packages/fluent/scss/card/_variables.scss:141]: The variable name is $kendo-card-subtitle-margin-bottom -> please update its description

File: packages/fluent/scss/card/_variables.scss [hunk 17]
```diff
@@ -2,176 +2,180 @@
...
 /// @group card
 $kendo-card-subtitle-margin-bottom: map.get( $kendo-spacing, 3 ) !default;
-/// Text color of the card subtitle.
+/// The letter spacing of the Card subtitle.
```
Comment (inikolova) [inline packages/fluent/scss/card/_variables.scss:144]: The variable name is $kendo-card-subtitle-text -> please update its description

File: packages/fluent/scss/card/_variables.scss [hunk 18]
```diff
@@ -2,176 +2,180 @@
...
+/// The shadow of the focused Card.
 /// @group card
 $kendo-card-focus-shadow: var( --kendo-box-shadow-depth-3, none ) !default;
 
-/// Text size of the card header.
+/// The font size of the Card header.
 /// @group card
 $kendo-card-header-font-size: var( --kendo-font-size-lg, inherit ) !default;
-/// Font family of the card header text.
+/// the font family of the Card header text.
```
Comment (inikolova) [inline packages/fluent/scss/card/_variables.scss:63]: Make the first letter uppercase and remove the word text. -> The font family of the Card header.

File: packages/fluent/scss/card/_variables.scss [hunk 19]
```diff
@@ -2,176 +2,180 @@
...
 /// @group card
 $kendo-card-header-font-family: var( --kendo-font-family, inherit ) !default;
-/// Text weight of the card header.
+/// The font weight of the Card header.
 /// @group card
 $kendo-card-header-font-weight: var( --kendo-font-weight, initial ) !default;
-/// Line height of the card header.
+/// The height of the Card header.
```
Comment (inikolova) [inline packages/fluent/scss/card/_variables.scss:69]: The line height...

**Review by inikolova — APPROVED** [review]

**Pull request merged**